tests/test_models.py

```diff
@@ -83,3 +83,19 @@ def test_mlp_classifier():
     model = MLPClassifier(input_size, 10000)
     model.fit(X, y)
     assert time.time() - start_time < 3, "Didn't early stop"
+    # Test with no positive examples.
+    y = np.zeros(len(X))
+    model = MLPClassifier(input_size, 100)
```

Comment:
Can we make these 100s bigger, and do a time check like above? (`assert time.time() - start_time < 3, "Didn't early stop"`) The time should be nearly instantaneous for the calls to `fit`.